_pages/install/frontend/building-the-spartacus-storefront-from-libraries-4-x.md

```diff
@@ -79,7 +79,7 @@ When you run this command, you are asked to choose which features you would like
 - To install the latest "next" release, you can add `@next` at the end of the command.
 - To install the latest release candidate, you can add `@rc` at the end of the command.
 - To verify what versions of Spartacus libraries were installed, open `package.json` and look for `@spartacus`.
-- To bypass the schematics prompts and install Spartacus with a predefined set of features, you can use the `--interactive=false` flag. For more information, see [Adding Spartacus Core Libraries and Features to Your Angular Project]({{ site.baseurl }}/schematics/#adding-spartacus-core-libraries-and-features-to-your-angular-project).
+- To bypass the schematics prompts and install Spartacus with a predefined set of features, you can use the `--no-interactive` flag. For more information, see [Adding Spartacus Core Libraries and Features to Your Angular Project]({{ site.baseurl }}/schematics/#adding-spartacus-core-libraries-and-features-to-your-angular-project).
```

Comment:
Revert changes in this file. It's doc for an older version of Spartacus.

Although it is not in one of the archived doc site version, the page says it's for spartacus 4

@gladius-mtl The pattern of these doc pages suggests that a 5.0 version of this file will be created for 5.0 and that the current 4.0 will be kept as-is somewhere.  The change in this file is relevant only for 5.0.  Not sure what to do here.

Reply:
@gladius-mtl Will create a 5.0 version of this file for us to update.